src/test/java/com/tryCloud/runners/CukesRunner.java

```diff
@@ -16,7 +16,7 @@
         features = "src/test/resources/features",
         glue = "com/tryCloud/step_definitions",
         dryRun = false,
-        tags = "@Deck"
+        tags = "@wip"
```

Comment:
i saw that you have just changed tag's value in order to run your feature file scenerios